wrappers/pkcs11/pkcs11_client.go

```diff
@@ -479,74 +570,177 @@ func (c *Pkcs11Client) CloseSession(session pkcs11.SessionHandle) {
 	c.client.CloseSession(session)
 }
 
-// Find on key for the given Label, ID and Mechanism.
-func (c *Pkcs11Client) FindKey(session pkcs11.SessionHandle, key Pkcs11Key, typ uint) (pkcs11.ObjectHandle, error) {
+func (c *Pkcs11Client) ExportRSAPublicKey(sh pkcs11.SessionHandle, key pkcs11.ObjectHandle) (*rsa.PublicKey, error) {
 	template := []*pkcs11.Attribute{
-		pkcs11.NewAttribute(pkcs11.CKA_LABEL, []byte(key.label)),
-		pkcs11.NewAttribute(typ, true),
+		pkcs11.NewAttribute(pkcs11.CKA_MODULUS, nil),
+		pkcs11.NewAttribute(pkcs11.CKA_PUBLIC_EXPONENT, nil),
+	}
+	attrs, err := c.client.GetAttributeValue(sh, key, template)
+	if err != nil {
+		return nil, fmt.Errorf("failed to pkcs#11 GetAttributeValue: %w", err)
 	}
-	if keyIdBytes, err := hex.DecodeString(key.id); err == nil {
+
+	var n = new(big.Int)
+	n.SetBytes(attrs[0].Value)
+	var e = new(big.Int)
+	e.SetBytes(attrs[1].Value)
```

Comment:
```suggestion
	var n = new(big.Int).SetBytes(attrs[0].Value)
	var e = new(big.Int).SetBytes(attrs[1].Value)
```

just to keep it compact